docs/source/api/v1/profiles_id_unassigned_parameters.rst

```diff
@@ -20,6 +20,8 @@
 *****************************************
 .. warning:: There are **very** few good reasons to use this endpoint - be sure not limit said use.
 
+.. deprecated::ATCv4
```

Comment:
i'm not seeing the deprecated label when i build the docs. does it need to look like this?

```.. deprecated:: ATCv4
	This endpoint and its functionality is deprecated, and will be removed in the future.
```

Reply:
yes, you need the space there